docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
 you must set *overwrite = false*. If you set overwrite to false, you may want to consider a cleanup function to ensure 
 you don't have 52 copies of bronze per year (assuming 1X / week).
 
+### Snapshot Process
+Snapshoting in Big Data World is very rare but often times the cluster logs and / or
+the audit logs are transient (especially Azure deployments as Event Hub only maintains 7 days). This means that if
+something happened to the bronze data your Overwatch history could be lost forever. To guard against this it's
+strongly recommended that you periodically snapshot the Overwatch data.
+
+With 0.7.2.1 release this snapshot would be done by Snapshot Class.
+
+Currently, There are two types of Snapshot process available in Overwatch:
+
+**Batch Snapshot**
+
+**Incremental Snapshot** 
+
+#### Batch Snaphot
+This is normal Snapshot process where we take the backup of Bronze in Batch method. Here we can do Deep Cloning of existing Bronze table to target Snapshot RootPath.
+
+#### Incremental Snap
+Through Incremental_Snap we take the snapshot of the bronze table incrementally i.e. through streaming operation.
+As described before as this is a streaming process we have to maintain a checkpoint directory. Below is the screenshot of the snap location after snapshot process is done:
```

Comment:
"This process will require us to maintain a checkpoint directory. Below is the screenshot of the snap location after snapshot process is done:"

Reply:
Done